packages/footer/components/Navigation.tsx

```diff
@@ -1,18 +1,102 @@
 import * as React from 'react';
 import {
    Box,
-   BoxProps,
    forwardRef,
    List,
    ListIcon,
    ListItem,
-   ListItemProps,
    ListProps,
    SimpleGrid,
+   Text,
+} from '@chakra-ui/react';
+import type {
+   BoxProps,
+   ListItemProps,
    SimpleGridProps,
 } from '@chakra-ui/react';
+import { NewsletterComponent } from './Newsletter';
+import type { NewsletterFormProps } from './Newsletter';
+import type { Menu as MenuType, MenuItem } from '@ifixit/ui';
 import { useTrackedOnClick } from '../hooks/useTrackedOnClick';
 
+type NavSectionProps = {
+   menu1: MenuType | null;
+   menu2: MenuType | null;
+   menu3: MenuType | null;
+   newsletterForm: NewsletterFormProps;
+};
+
+export const NavigationSection = ({
+   menu1,
+   menu2,
+   menu3,
+   newsletterForm,
+}: NavSectionProps) => {
+   return (
+      <FooterNavigationSection p={5}>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu1} />
+         </FooterNavigationList>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu2} />
+         </FooterNavigationList>
+         <FooterNavigationList pl={-5}>
+            <NavigationColumn menu={menu3} />
+         </FooterNavigationList>
+         <NewsletterComponent newsletterForm={newsletterForm} />
+      </FooterNavigationSection>
+   );
+};
+
+const NavigationColumn = ({ menu }: { menu: MenuType | null }) => {
+   console.log('Menu: ', menu);
```

Comment:
Is this `console.log` supposed to be here?

Reply:
Nope! Great find. Removed in [136721d](https://github.com/iFixit/react-commerce/pull/1342/commits/136721df9799b9857110515151d1b26bfb7f3ab4).